defaults/main.yml

```diff
@@ -31,5 +31,8 @@ apache_htpasswd_file:
 apache_htpasswd_bypass_from: "127.0.0.1"
 
 # Whether PHPmyadmin should be installed.
-# By default, disable it for PHP7 as the packages are incompatible.
-lamp_phpmyadmin: "{{ true if php_version < 7 else false }}"
\ No newline at end of file
+lamp_adminer: true
+
+
+# Provide backwards compatibility with old projects.
+phpmyadmin_htpasswd: "{{ adminer_htpasswd }}"
```

Comment:
There is no BC if nothing uses phpmyadmin_htpasswd. Nevermind, I do not hink we have to keep BC here.